### ISSUE TITLE
Ships occasionally won't align to maneuver nodes

### ISSUE THREAD

**Issue by SpareBuffalo** [MuMech/MechJeb2#959]
Very occasionally a ship won't align either via MJ or via the stock autopilot. The angle in both cases I have experienced has been off by only a couple of degrees. In MJ the error is greater than the stock, I have never experienced an error of more than ten degrees. This results in rotation about an odd axis, and MJ won't auto-warp or execute the node. It is verified fixed by returning to main menu. 

The photo shows the alignment error. The craft was rotating around the maneuver node axis without pointing the nose straight into the maneuver node.
![wont align](https://user-images.githubusercontent.com/34152813/33470927-7cee0090-d630-11e7-8a91-cbc5dc9503f9.PNG)

**Comment by brianmcnett** [comment]
I appear to experience this as well, also with the result that auto-warp
does not engage when this occurs. I've found that manually rotating the
craft to 0° roll will momentarily restore alignment usually enough to
engage auto-warp. This remains true for build #762 although the error
appears smaller than shown. I haven't experienced the node-execution
failure. I'm attempting to reproduce for the purposes of collecting logs.

**Comment by lamont-granquist** [comment]
In the Attitude Adjustment controller try using the "HybridController" and in general it seems the PID needs more Integral and less deadband (i think).

**Comment by SpareBuffalo** [comment]
Okay, well I haven't had it happen since I posted this. Couldn't find any setting about HybridController, but I changed the other two settings. Pardon my ignorance, but I'm not at all sure what they changed lol. Don't even know what Deadband (have an idea) or Integral are (math 🙄).

**Comment by brianmcnett** [comment]
I find that switching to HybridController frequently does nothing at all
(I've had this work all of one time and could not recreate the conditions).
I tried reducing the deadband, but it is already down in the thousandths. I
will admit to being fuzzy about the workings of PID controllers generally.
but presuming the P, I, and D are represented by Kp, Ki, and Kd
respectively... Ki is the integral.  What are good limits for that value?

**Comment by lamont-granquist** [comment]
Yeah, no tweaking is working for me either, but I haven't gotten the time to Debug.Log what is actually going on

**Comment by lamont-granquist** [comment]
manually increasing warp with '.' should work BTW, its just annoying.

**Comment by brianmcnett** [comment]
Manually toggling warp on, and immediately back off again does work.
Autowarp will then engage every time. Annoying is right.

**Comment by lamont-granquist** [comment]
```
                if ((core.attitude.attitudeAngleFromTarget() < 1 && core.vessel.angularMomentum.magnitude < 0.05) || (core.attitude.attitudeAngleFromTarget() < 10 && !MuUtils.PhysicsRunning()))
```

its almost certainly that line in the NodeExecutor.  it should probably use angular velocity instead of momentum.   dunno if you can find that off of Vessel or off of MJ's vesselState already, or if you need to just divide by mass there, and offhand IDK what a good value would be.
